src/Compilers/Core/MSBuildTaskTests/CscTests.cs

```diff
@@ -659,5 +659,49 @@ public void StdLib()
 
             AssertEx.Equal("/out:test.exe test.cs", csc.GenerateResponseFileContents());
         }
+
+        [ConditionalFact(typeof(UnixLikeOnly))]
```

Comment:
Add WorkItem attributes to the new tests, referencing the issue (https://github.com/dotnet/roslyn/issues/80865).